.gitignore

```diff
@@ -1,3 +1,90 @@
-node_modules
-/community-connectors/meetup/BUILD
```

Comment:
Probably you want to preserve this?

Reply:
This actually shouldn't have been in there in the first place.